requirements.txt

```diff
@@ -1 +1,13 @@
+aws_cdk.aws_apigateway
+aws_cdk.aws_cognito
+aws_cdk.aws_iam
+aws_cdk.aws_kinesisfirehose
+aws_cdk.aws_kms
+aws_cdk.aws_logs
+aws_cdk.aws_pinpoint
+aws-cdk.aws_s3
+aws_cdk.aws_ssm
+aws_cdk.core
 black
+boto3
+isort
```

Comment:
Is this required because we have a pre-commit hook or something?

Reply:
No precommit hook yet, but I'm planning on doing a reformat and updating the contribution guide with code style guidelines